dependencies.xml

```diff
@@ -64,8 +64,13 @@ Note all install methods after "main" take
     <line_profiler optional='True'/>
     <!-- <ete3 optional='True'/> -->
     <pywavelets optional='True'>1.1</pywavelets>
+    <numdifftools source="pip">0.9.39</numdifftools>
```

Comment:
Hm, do we need to specify the 0.9.39, or is 0.9 sufficient?

Reply:
I think it would be safe to stick with 0.9.39 for now.